_posts/2019-09-11-how-to-stay-up-to-date.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: How to stay up to date with your Rails application
+date: 2019-08-27 13:56:00
+categories: ["Rails", "Upgrade"]
```

Comment:
@bronzdoc Please try to use the same categories we have been using for other similar articles. It should be "Upgrades"